transport/grpc/handler.go

```diff
@@ -98,6 +114,35 @@ func getServiceAndProcedureFromMethod(streamMethod string) (string, string, erro
 	return service, procedure, nil
 }
 
+func getContextMetadata(ctx context.Context) (metadata.MD, error) {
+	contextMetadata, ok := metadata.FromContext(ctx)
+	if !ok || contextMetadata == nil {
+		return nil, errors.New("Could not extract metadata information from context")
```

Comment:
error messages should start with lower case

https://github.com/golang/go/wiki/CodeReviewComments#error-strings